### PR TITLE
chore: Fix build issues in "packages/playground"

### PR DESCRIPTION
**Pull request by vardanbansal-harness**

### CODE REVIEW

**Review by cjlee01 — APPROVED** [review]

File: packages/playground/src/pages/landing-page.tsx
Comment (jessieyu777) [inline packages/playground/src/pages/landing-page.tsx:1]: It seems like we can use noop from lodash for remaining potential props with api call in the future?

File: packages/playground/src/pages/create-project-page.tsx
Comment (jessieyu777) [inline packages/playground/src/pages/create-project-page.tsx:1]: Thanks for letting me know. I will delete the temperate comment in the future

File: packages/playground/src/pages/create-pipeline-page.tsx
Comment (jessieyu777) [inline packages/playground/src/pages/create-pipeline-page.tsx:1]: Do we have two icon systems, one is logoName and another one is IconName? I just implemented Icon before but logoName not yet so curious about it.

File: packages/playground/src/pages/landing-page.tsx
Comment (vardanbansal-harness) [reply packages/playground/src/pages/landing-page.tsx:1]: Yeah you can use it instead of passing `() => {}`

**Pull request merged**